Validation/MuonCSCDigis/BuildFile.xml

```diff
@@ -1,15 +1,20 @@
-<use   name="DQMServices/Core"/>
 <use   name="DataFormats/CSCDigi"/>
 <use   name="DataFormats/Common"/>
+<use   name="DataFormats/GEMDigi"/>
+<use   name="DataFormats/MuonDetId"/>
+<use   name="DQMServices/Core"/>
 <use   name="FWCore/Framework"/>
 <use   name="FWCore/MessageLogger"/>
 <use   name="FWCore/ParameterSet"/>
 <use   name="FWCore/ServiceRegistry"/>
 <use   name="Geometry/CSCGeometry"/>
+<use   name="Geometry/GEMGeometry"/>
```

Comment:
Could you check if DataFormats/GEMDigi, Geometry/GEMGeometry, DataFormats/MuonDetId are really needed?

Reply:
It would appear they are not.